sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -195,7 +195,7 @@ def __init__(self,
     self._state = None
     self._state_queues = []
     self._log_queues = []
-    self.state = beam_job_api_pb2.JobState.STARTING
+    self.state = beam_job_api_pb2.JobState.STOPPED
```

Comment:
Why this change?

Reply:
The initial state in java runners is STOPPED.  Then it transitions to STARTING and RUNNING.  More info at #9969.